app/jobs/fetch_globus_job.rb

```diff
@@ -26,6 +28,10 @@ def globus_prefix(work_version)
     "#{Settings.globus.uploads_directory}#{work_version.globus_endpoint}/"
   end
 
+  def ignore?(path)
+    path.start_with?('__MACOSX') || path.end_with?('.DS_Store')
```

Comment:
I was a bit confused why you wouldn't want to ignore` __MACOSX` anywhere in the path.

Reply:
@edsu Good question! I'm not sure where to expect `__MACOSX` subdirectories, so I opted for parity w/ what the `UnzipJob` is currently doing: https://github.com/sul-dlss/happy-heron/blob/main/app/jobs/unzip_job.rb#L33